vault/version_store.go

```diff
@@ -7,12 +7,15 @@ import (
 	"strings"
 	"time"
 
+	semver "github.com/hashicorp/go-version"
 	"github.com/hashicorp/vault/sdk/helper/consts"
 	"github.com/hashicorp/vault/sdk/logical"
+	"github.com/hashicorp/vault/version"
 )
 
 const (
 	vaultVersionPath string = "core/versions/"
+	lastUpgradeKey   string = "core/last_upgrade_version/"
```

Comment:
Remember that you'll want an ent PR as well to flag this as an ignored prefix.  Also, why the trailing slash?